tests/time_tracker_api/time_entries/time_entries_query_builder_test.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from unittest.mock import patch
+from utils.query_builder import CosmosDBQueryBuilder
+from time_tracker_api.time_entries.time_entries_query_builder import (
+    TimeEntryQueryBuilder,
+)
+from utils.repository import remove_white_spaces
+
+
+def test_TimeEntryQueryBuilder_is_subclass_CosmosDBQueryBuilder():
+    query_builder = CosmosDBQueryBuilder()
+    time_entries_query_builder = TimeEntryQueryBuilder()
+
+    assert issubclass(TimeEntryQueryBuilder, CosmosDBQueryBuilder) == True
+
+
+@pytest.mark.parametrize(
+    "start_date,end_date,expected_params",
+    [
+        (
+            "2021-03-19T05:07:00.000Z",
+            "2021-03-25T10:00:00.000Z",
+            [
+                {"name": "@start_date", "value": "2021-03-19T05:07:00.000Z"},
+                {"name": "@end_date", "value": "2021-03-25T10:00:00.000Z"},
+            ],
+        ),
+        (
+            "2021-02-10T05:08:00.000Z",
```

Comment:
I have the doubt if the three scenarios are revelant. We may only choose one.

Reply:
It could be only one, they are not relevant cases